<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Morphology</class>
 <widget class="QMainWindow" name="Morphology">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Morphology</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>321</width>
      <height>47</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_type">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_kernel_type">
       <property name="text">
        <string>Kernel Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_kernel_size">
       <property name="text">
        <string>Kernel Size</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QComboBox" name="comboBox_type">
       <item>
        <property name="text">
         <string>Erosion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dilation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Open</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Close</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gradient</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TopHat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>BottomHat</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_2_kernel_type">
       <item>
        <property name="text">
         <string>Square</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Circle</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cross</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QComboBox" name="comboBox_kernel_size">
       <item>
        <property name="text">
         <string>(3×3)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(5×5)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(7×7)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>(9×9)</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>195</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Morphology</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>104</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
